### PR TITLE
chore: sentence case all cli options description

### PR DESCRIPTION
**Pull request by dammy001**

### CODE REVIEW

File: packages/vitest/src/node/cli.ts [hunk 1]
```diff
@@ -10,24 +10,24 @@ const cli = cac('vitest')
 
 cli
   .version(version)
-  .option('-r, --root <path>', 'root path')
-  .option('-c, --config <path>', 'path to config file')
-  .option('-u, --update', 'update snapshot')
-  .option('-w, --watch', 'watch mode')
-  .option('-t, --testNamePattern <pattern>', 'run test names with the specified pattern')
-  .option('--ui', 'open UI')
+  .option('-r, --root <path>', 'Root path')
```
Comment (antfu) [inline packages/vitest/src/node/cli.ts:13]: Emmm, I am not so sure about the case change here

Comment (antfu) [reply packages/vitest/src/node/cli.ts:13]:
```bash
➜ git commit -h                     
usage: git commit [<options>] [--] <pathspec>...

    -q, --quiet           suppress summary after successful commit
    -v, --verbose         show diff in commit message template

Commit message options
    -F, --file <file>     read message from file
    --author <author>     override author for commit
    --date <date>         override date for commit
    -m, --message <message>
                          commit message
    -c, --reedit-message <commit>
                          reuse and edit message from specified commit
    -C, --reuse-message <commit>
                          reuse message from specified commit
    --fixup <commit>      use autosquash formatted message to fixup specified commit
    --squash <commit>     use autosquash formatted message to squash specified commit
    --reset-author        the commit is authored by me now (used with -C/-c/--amend)
    -s, --signoff         add a Signed-off-by trailer
    -t, --template <file>
                          use specified template file
    -e, --edit            force edit of commit
    --cleanup <mode>      how to strip spaces and #comments from message
    --status              include status in commit message template
    -S, --gpg-sign[=<key-id>]
                          GPG sign commit
```

It seem more common to use lowercase in CLI, let's keep the original then.

Comment (dammy001) [reply packages/vitest/src/node/cli.ts:13]: Then we'll have to change all to lowercase because it wasn't consistent before

**Comment by dammy001** [comment]
@antfu 👀👀

**Review by patak-dev — APPROVED** [review]

**Pull request merged**